plugins/view/dummyView/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
note that this directory can be located anywhere - it doesn't have to be in DIGITS tree